pages/tidal tools/analyze_database.md

```diff
@@ -225,3 +225,13 @@ databases:
 {% include note.html content="By setting this to `false`, some results from the
 analysis will not be available, including server metrics, connected applications,
 and the workload analysis ranking." %}
+
+## Troubleshooting
+
+If you are getting errors in tring to perform the analysis it can help if you confirm that you do have network connectivity to the database.
```

Comment:
```suggestion
If you are getting errors in tring to perform the analysis, it can help if you confirm that you do have network connectivity to the database.
```